ToNote/Views/EditNoteDialogView.xaml.cs

```diff
@@ -1,4 +1,7 @@
-﻿namespace ToNote.Views
```

Comment:
Is this view used anywhere? If not, remove it. If it was left "Because we might need it later", we can get it from git history if we need it later. Same for related cases.